ml/base.py

```diff
@@ -262,8 +263,8 @@ def initialize_input_transform(self, x, transform=True, overwrite=True):
             self.x_scaling_mins = np.zeros(n_parameters)
             self.x_scaling_maxs = np.ones(n_parameters)
 
-    def _transform_inputs(self, x, scaling = "minmax"):
-        if scaling == "standard":    
+    def _transform_inputs(self, x, scaling = "standard"):
```

Comment:
@neko-0 

We should not put this as default, but rather the min-max as default.